repos/system_upgrade/common/actors/ipuworkflowconfig/actor.py

```diff
@@ -18,8 +19,8 @@ class IPUWorkflowConfig(Actor):
     tags = (IPUWorkflowTag,)
 
     def process(self):
-        flavour = ipuworkflowconfig.get_upgrade_flavour()
-        target_version = ipuworkflowconfig.get_target_version(flavour)
+        flavour = upgrade_path.get_upgrade_flavour()
```

Comment:
@pirat89 Wondering if I should move all code from process into the library and leave here just a single func call. WDYT?